variant.go

```diff
@@ -111,3 +116,39 @@ func (s *VariantServiceOp) Update(variant Variant) (*Variant, error) {
 func (s *VariantServiceOp) Delete(productID int64, variantID int64) error {
 	return s.client.Delete(fmt.Sprintf("%s/%s/%d/variants/%d.json", globalApiPathPrefix, productsBasePath, productID, variantID))
 }
+
+// List metafields for a variant
```

Comment:
Go docstring convention is to begin the comment with the name of the element being described. https://blog.golang.org/godoc-documenting-go-code I know we don't always follow that convention in this library but I think we should try to stick to it going forward.

Reply:
I updated all the comments to the right convention in both the variant and product files